Tests/LegacyUnit/Service/RegistrationManagerTest.php

```diff
@@ -6387,10 +6387,14 @@ public function createRegistrationIncreasesRegistrationCountInEventFromZeroToOne
 
         $subject->createRegistration($this->seminar, [], $plugin);
 
-        $seminarData = \Tx_Oelib_Db::selectSingle('*', 'tx_seminars_seminars', 'uid = ' . $this->seminarUid);
+        /** @var ConnectionPool $connectionPool */
+        $connectionPool = GeneralUtility::makeInstance(ConnectionPool::class);
+        $connection = $connectionPool->getConnectionForTable('tx_seminars_attendances');
```

Comment:
We need two separate connections here - one for `tx_seminars_attendances` and one for `tx_seminars_seminars`.